integrations/cypress/README.md

```diff
@@ -1,17 +1,32 @@
 # @simulacrum/auth0-cypress
 
-[Cypress](https://www.cypress.io/) [Auth0](https://auth0.com) addon that simulates an auth0 server running on `localhost` so you don't have to create fake accounts while developing or running tests that require authentication.
+A [Cypress](https://www.cypress.io/) plugin specifically created for SDK's that integrate with [Auth0](https://auth0.com)! @simulacrum/auth0-cypress will simulate an auth0 server, running on `localhost`, so you don't have to create fake Auth0 accounts while developing or running tests that require authentication!
```

Comment:
> 1. Is the `mkcert` command needed to make the encrypt task work when configuring Cypress?

The encrypt task used to be needed to create the auth0 cookie.  I think you no longer do that.  Why is that.

> 2. Is the `mkcert` needed because the Auth0Simulation server runs on an https localhost domain?

Yes

> 3. "Should" this Cypress plugin work when running in a CI/CD pipeline? Maybe add some instructions for that as well if it should work?

Not something I've ever done

> 4. Is it maybe better to add this as a step 6 under "Installation" when setting everything up instead of the project description?

That seems better

Reply:
> The encrypt task used to be needed to create the auth0 cookie. I think you no longer do that. Why is that.

I have some trouble remembering, but I think I removed it due to the command, in how it was used here, being [deprecated](https://docs.cypress.io/api/cypress-api/cookies) (see history at the bottom) but I might not have properly replaced it with `cy.origin()` or `cy.session()`. That is also something on my todo list on how that should work in the newer Cypress versions

I left the encrypt task in it, but it indeed might be removable due to the migration. Is it okay if I create some issues with these points which I will pick up in separate PR's to keep it better organized?

Also for the `mkcert` point that the README should reflect it better

Reply:
I have added a description with a link as a step 6 to the README :+1: